core/spec.md

```diff
@@ -3190,36 +3167,68 @@ Clarifying the  usage of the `attributes`, `resourceattributes` and
 
 #### Retrieving the Registry Model
 
-To retrieve the Registry Model as a stand-alone entity, an HTTP `GET` MAY be
-used.
+The Registry model is available in two forms:
+- The full "model" with all possible aspects of the model defined.
+- The "modelsource" form represents just the model aspects specified when
+  the model was defined.
+
+The full "model" view can be thought of as a full schema definition of what the
+messages exchanges with the server might look like. As such, it'll include
```

Comment:
```suggestion
messages exchanges with the server might look like. As such, it includes
```

Reply:
(Here and below, the future tense feels a bit odd.)

Reply:
done - both